scipy/spatial/tests/test_distance.py

```diff
@@ -1626,7 +1626,12 @@ def test_cosine(self):
         for x, y in self.cases:
             dist = wcosine(x, y)
             assert_almost_equal(dist, 1.0 - 18.0 / (np.sqrt(14) * np.sqrt(27)))
-
+    
+    def test_cosine_output_dtype(self):
+        # Regression test for gh-19541
+        assert_(isinstance(wcorrelation([1, 1], [1, 1], centered=False), float))
+        assert_(isinstance(wcosine([1, 1], [1, 1]), float))
+      
```

Comment:
Small nit, some whitespace has crept in. Also, we have a preference for `assert` over `numpy.testing.assert_`.
```suggestion

    def test_cosine_output_dtype(self):
        # Regression test for gh-19541
        assert isinstance(wcorrelation([1, 1], [1, 1], centered=False), float))
        assert isinstance(wcosine([1, 1], [1, 1]), float))

```
After this, looks good to me if CI is green!